src/lib/extract/extractFilter.ts

```diff
@@ -53,3 +55,27 @@ export const extractFeColorMatrix = (
 
   return extracted;
 };
+
+export const extractFeGaussianBlur = (
+  props: FeGaussianBlurComponentProps
+): FeGaussianBlurNativeProps => {
+  const extracted: FeGaussianBlurNativeProps = {};
+
+  if (props.in) extracted.in1 = props.in;
+  if (props.stdDeviation !== undefined) {
+    if (
```

Comment:
Once again, is splitting those values part of the standard?

Reply:
Yes, according to W3 specs https://www.w3.org/TR/SVG11/filters.html#feGaussianBlurStdDeviationAttribute

> stdDeviation = "[number-optional-number](https://www.w3.org/TR/SVG11/types.html#DataTypeNumberOptionalNumber)"

> The standard deviation for the blur operation. If two [number](https://www.w3.org/TR/SVG11/types.html#DataTypeNumber)s are provided, the first number represents a standard deviation value along the x-axis of the coordinate system established by attribute [‘primitiveUnits’](https://www.w3.org/TR/SVG11/filters.html#FilterElementPrimitiveUnitsAttribute) on the [‘filter’](https://www.w3.org/TR/SVG11/filters.html#FilterElement) element. The second value represents a standard deviation in Y. If one number is provided, then that value is used for both X and Y.
...
A value of zero disables the effect of the given filter primitive (i.e., the result is the filter input image). If [‘stdDeviation’](https://www.w3.org/TR/SVG11/filters.html#feGaussianBlurStdDeviationAttribute) is 0 in only one of X or Y, then the effect is that the blur is only applied in the direction that has a non-zero value.